src/explore-education-statistics-admin/src/pages/release/pre-release/PreReleasePageContainer.tsx

```diff
@@ -104,6 +104,18 @@ const PreReleasePageContainer = ({
     }
 
     if (access === 'Before') {
+      const zonedStartDate = utcToZonedTime(start, 'Europe/Berlin');
```

Comment:
Why Berlin not London here?

Reply:
Because I forgot to remove that line whilst playing around with this func during development 🙈 Will remove!